src/atomics/multithread.md

```diff
@@ -0,0 +1,220 @@
+# Multithreaded Execution
+
+The first important thing to understand about C++20 atomics is that **the
+abstract machine has no concept of time**. You might expect there to be a single
+global ordering of events across the program where each happens at the same time
+or one after the other, but under the abstract model no such ordering exists;
+instead, a possible execution of the program must be treated as a single event
+that happens instantaneously — there is never any such thing as “now”, or a
+“latest value”, and using that terminology will only lead you to more confusion.
+(Of course, in reality there does exist a concept of time, but you must keep in
+mind that you’re not programming for the hardware, you’re programming for the
+AM.)
```

Comment:
This acronym is never defined, maybe add "(AM)" the first time you mention "abstract machine/model," or just type it in full here.

Reply:
I’ve introduced AM as an acronym in the intro (`atomics.md`), which should be enough.